qiita_db/test/test_processing_job.py

```diff
@@ -22,7 +22,7 @@
 from qiita_core.qiita_settings import qiita_config
 
 
-def _create_job():
+def _create_job(force=True):
```

Comment:
If we default to `True`, are there situations in the codebase where this is `False`?

Reply:
Or rather, this API has changed I believe since the existing functionality would be `force=False` if I understand the docs above. Will this change be a problem elsewhere?

Reply:
...or was the default behavior previously to force the execution?